org.eclipse.winery.topologymodeler.ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "org.eclipse.winery.topologymodeler.ui",
-    "version": "2.0.0-SNAPSHOT",
+    "name": "topologymodeler",
```

Comment:
I would use `winery-topologymodeler`

Reply:
I will leave the original name "org.eclipse.winery.topologymodeler.ui" so it is consistent with the ui and workflowmodeler names